### ISSUE TITLE
Create an FAQ for contributors

### ISSUE THREAD

**Issue by hhkaos** [EsriDevEvents/contributor-guides#2]
As well as [requirements](https://github.com/EsriDevEvents/contributor-guides/issues/1), I think there are some questions that we should address and clarify on a CONTRIBUTING.md. For example:

* What if I did the same or very similar talk with a different name?
   * Should we create a new repository? <- yes, you can add the year at the end (`session-name-year`)
      * Won't it be too messy? (we will try to keep previous repos archived)
   * Should we use the same repository with a new folder per edition? <- yes if you are using gh-pages and need to keep several editions accesible.
   * Should we use the same repository and create a new branch per edition or [create a tag](https://docs.github.com/en/rest/git/tags?apiVersion=2022-11-28)? <- this could be another alternative
* Should I keep old topics when updating the content? (e.g. year & edition) <- yes
* I don't have access to an existing repository, how do I get access? create and issue and ask the maintainer permissions or contact DeveloperOutreach@esri.com
* What if I prefer to host these repositories in my personal GitHub account?
   * Should we fork these repositories?
* What if I reuse most of the content from an existing repo?
* When should a repo be archived?

**Comment by hhkaos** [comment]
> What if I did the same or very similar talk with a different name?

I think if the name is different a new repository should be created, but a cross-link referencing both repositories would be desirable.

If the name is the same one I would suggest [tagging the commit](https://git-scm.com/book/en/v2/Git-Basics-Tagging) of the final version that was delivered at that specific event and make the necessary changes to the repo for the latest version.

@tzrh @gavinr does it makes sense?

I see a clear benefit of having only one repository per session. I think it makes sense for people interested on the topic to star/follow a repo to be notified when new content is uploaded in future version.

What do you think?

**Comment by tzrh** [comment]
One issue with reusing the repository is that any content hosted on github pages still needs to be split by year. e.g. [this](https://github.com/EsriDevEvents/arcgis-js-api-for-javascript-3d-visualization/) 2022 session uses reveal.js and so could be made available [online](https://esridevevents.github.io/arcgis-js-api-for-javascript-3d-visualization/) (although it looks broken at the moment). We'd like to do the same this year, but would then need to add an extra directory for the new session e.g. `https://esridevevents.github.io/arcgis-js-api-for-javascript-3d-visualization/2023/`

The other thing to think about would be how new presenters would gain access to the previous year's repository. Would they need to contact the previous year's presenters to be added as contributors, or is that something that a single point of contact could do?

**Comment by hhkaos** [comment]
You are right @tzrh, for those using gh-pages makes sense to use a folder.

> session uses reveal.js and so could be made available online (although it looks broken at the moment)

Yes, I broke it, I'm going to fix it asap.

> The other thing to think about would be how **new presenters would gain access to the previous year's repository**. Would they need to contact the previous year's presenters to be added as contributors, or is that something that a single point of contact could do?

**Update**: Sorry I didn't read the "new". You can open an issue in the repo and ask the previous presenter and/or @Kneeson, @cyatteau or myself and we will give you access permissions.

The repository is still the same, we have just renamed the repos, so I assume presenters still have access to them, let me know if I'm wrong. A

**Comment by hhkaos** [comment]
@tzrh I'm talking to @gavinr and for simplicity (for now) we have decided to allow also using a new repository for this year adding the year in the title (e.g. https://github.com/EsriDevEvents/arcgis-experience-builder-customizing-and-extending-2023)
